README.md

```diff
@@ -1,22 +1,42 @@
+# Env variables
+```
+API_PORT={number, ex.3040}
+
+LANG={RUST|GO|NODE|JAVA}
+ETH_NODE_URL={infura or alchemy endpoint}
+DB_CONNECTION_URL={postgres connection string}
+INTEGRATED=
+```
+
 # Using the test tool
 
+## Set env variables
+```
+export API_PORT="8080"
+export DB_CONNECTION_URL="postgresql://localhost/postgres?user=postgres&password=password1234"
```

Comment:
postgres default password is postgres

Reply:
This is in line with running postgres using docker command from above i.e. `docker run --name postgres --network my-app -p 5432:5432 -e POSTGRES_PASSWORD=password1234 -d postgres`.
Looking at [the DockerHub page](https://hub.docker.com/_/postgres), it says:
```
The PostgreSQL image uses several environment variables which are easy to miss. The only variable required is POSTGRES_PASSWORD, the rest are optional.
```

So it seems like the password is required and there's no default value.
Perhaps when you say by "default" you mean "by convention"? Or is it default when run in some other fashion i.e. not with Docker?

Reply:
Yeah, I meant by convention. This is more of a nit comment, don't dive too deep into it.